MacOS CI fails

@grahambates https://github.com/BartmanAbyss/vscode-amiga-debug/actions/runs/7597862962/job/20693197771
```
configure: error: source highlight is incompatible with -static-libstdc++; either use --disable-source-highlight or --without-static-standard-libraries
```

--- a/ci/binutils-gdb/configure-macos.sh
+++ b/ci/binutils-gdb/configure-macos.sh
@@ -18,5 +18,6 @@ cd build-binutils-gdb
     --disable-werror \
 	--without-guile \
     --enable-static \
+	--disable-source-highlight \
     --prefix="$PREFIX" \
     --target=m68k-amiga-elf

